Automattic/wp-calypso
# Seller Experience: Add the Design Picker to the flow

#### Changes proposed in this Pull Request

This PR adds the Design Picker to the "Simple" seller flow.

* Update the `storeType` types to use the correct values (they were already set in the `types.ts` file as `simple` and `power`, so I'm using those instead of `pattern_block` and `woocommerce`, respectively)
* Remove payment block pattern setup and `isFSEActive` dependency from `store-features` API call, since we handle this in the design picker now.
* Skip design selection for the WooCommerce flow.

**Video demo**


https://user-images.githubusercontent.com/2124984/156245345-3eb6312d-ef6d-4ea6-b2d9-faa9ce591fb2.mov



#### Testing instructions

* Switch to this PR, navigate to `/start`, and create a new free site
* Select "Sell" at the intent step
* Select "Start simple" after setting your store title and tagline
* You should be brought to the design picker with the "Store" category marked "recommended", and five designs showcased (Marl, Dorna, Hari, Attar, and Winkel); choose a design.
* You should land in the site editor with your chosen design.
* Double-check that other intent flows and the Woo flow still work as expected.

Related to #61470
